src/helper/selection-tools/move-tool.js

```diff
@@ -97,7 +98,13 @@ class MoveTool {
         this.setSelectedItems();
     }
     onMouseDrag (event) {
-        const dragVector = event.point.subtract(event.downPoint);
+        const point = event.point;
+        if (event.point.x > ART_BOARD_WIDTH || event.point.y > ART_BOARD_HEIGHT ||
+                event.point.x < 0 || event.point.y < 0) {
+            point.x = Math.max(0, Math.min(point.x, ART_BOARD_WIDTH));
```

Comment:
Do you need the big if statement given that you're doing `Math.max` and `Math.min`?

Reply:
Oh, I suppose not